doc/api/stream.md

```diff
@@ -341,6 +341,23 @@ reader.pipe(writer);
 reader.unpipe(writer);
 ```
 
+##### writable.acquireStandardStream()
```

Comment:
This will definitely need more documentation about the new objects

Reply:
> Another thing to keep in mind is that the `readable-streams`module will need to be able to implement support for this at some point.

i would disagree that the readable streams module should do that. in any case, these added methods work with any global polyfill.

Reply:
I still don't really understand what the problem is here. is the point that we can't add any features to node core because the external module doesn't have those changes?

> nor will it be possible to implement those methods for every browser currently supported by readable-streams.

the module can just use feature detection like any other browser library.

Reply:
The point is this can be done in a way where this isn't even a problem by not extending the prototype.

Reply:
I'm hopeful we can come up with something that isn't importing separate functions. I want to have as little friction as possible for people wanting to use whatwg streams, as there are quite a lot of locations where the interop needs to happen. I'm willing to continue bikeshedding this for as long as we need to for something that works for everyone, but it would seem we both have constraints here.

Reply:
There are some relevant comments here that are being buried a bit after rebasing... just linking them here so that they don't get lost in the conversation: https://github.com/nodejs/node/pull/22352/commits/1ca4477123a852c0aa3bfec05aec8901bc251216#r211124305 and https://github.com/nodejs/node/pull/22352/commits/1ca4477123a852c0aa3bfec05aec8901bc251216#r211131273